eglib/src/gpath.c

```diff
@@ -250,11 +250,13 @@ g_find_program_in_path (const gchar *program)
 		
 		x = NULL;
 		probe_path = g_build_path (G_DIR_SEPARATOR_S, l, program, NULL);
+#ifdef X_OK
 		if (access (probe_path, X_OK) == 0){ /* FIXME: on windows this is just a read permissions test */
```

Comment:
I agree, this is an over-zealous check. I don't recall why we needed it when I originally did this, but it seems to be unnecessary on the platforms we care about now. I'll remove it.

Should I squash this new commit into the previous one and do a force push to this branch so that we only see one commit? Or is it acceptable to have two commits on this branch if/when it is merged?